package.json

```diff
@@ -32,13 +32,20 @@
     "format:check": "prettier --check \"**/*.+(json|yml|yaml|css|md)\"",
     "flow:check": "flow check",
     "concurrently": "concurrently",
-    "postinstall": "husky install"
+    "postinstall": "node -e \"process.env.NODE_ENV != 'production' && process.exit(1)\" || husky install"
   },
   "eslintConfig": {
+    "env": {
```

Comment:
not sure this is correct, but went by the the documentation

Reply:
This looks good to me. Tried to find if there's a node script for this case but couldn't find any.